src/functions/quadratic.jl

```diff
@@ -19,6 +19,7 @@ If `iterative=true`, then `prox!` is evaluated approximately using an iterative
 
 abstract type Quadratic <: ProximableFunction end
 
+is_convex(f::Quadratic) = true
```

Comment:
This I have not previously put because one should really check that `Q` is positive semidefinite...

Reply:
Any thoughts on this @mfalt ? Of course checking the eigenvalues is not viable, but maybe the user should be able to assert whether the function is convex or not?

Reply:
No ok, the docstring says we assume `Q` to be positive definite (should be semidefinite, to be corrected). We may want to change this in the near future, not all quadratics are convex!

Reply:
Yes, I added this because of the documentation, but you are right, maybe some keyword from the user, acknowledging if it is positive(semi)-definite would be good.